.github/workflows/release.yml

```diff
@@ -0,0 +1,134 @@
+        if: env.should-release == 'true'
+        working-directory: app
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "RELEASE_NOTES<<EOF" >> "$GITHUB_OUTPUT"
+          git cliff --include-path "app/**/*" --repository "../" --unreleased >> "$GITHUB_OUTPUT"
+          echo "EOF" >> "$GITHUB_OUTPUT"
+      - name: Update version
+        working-directory: app
+        if: env.should-release == 'true'
+        run: |
+          sed -i '' -e "s/CFBundleShortVersionString.*/CFBundleShortVersionString\": \"${{ steps.next-version.outputs.NEXT_VERSION_NUMBER }}\",/g" "Project.swift"
+          sed -i '' -e "s/CFBundleVersion.*/CFBundleVersion\": \"${{ steps.next-version.outputs.NEXT_VERSION_NUMBER }}\",/g" "Project.swift"
+      - name: Update CHANGELOG.md
+        working-directory: app
+        if: env.should-release == 'true'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: git cliff --include-path "app/**/*" --repository "../" --bump -o CHANGELOG.md
+      - name: Select Xcode
+        if: env.should-release == 'true'
+        run: sudo xcode-select -switch /Applications/Xcode_15.3.app
+      - name: Install dependencies
+        if: env.should-release == 'true'
+        run: mise run install
+      - name: Bundle Tuist App
+        if: env.should-release == 'true'
+        run: mise run bundle:app
+        env:
+            CERTIFICATE_PASSWORD: ${{ secrets.CERTIFICATE_PASSWORD }}
+            APPLE_ID: ${{ secrets.APPLE_ID }}
+            APP_SPECIFIC_PASSWORD: ${{ secrets.APP_SPECIFIC_PASSWORD }}
+            CERTIFICATE_ENCRYPTION_PASSWORD: ${{ secrets.CERTIFICATE_ENCRYPTION_PASSWORD }}
+            TUIST_CONFIG_TOKEN: ${{ secrets.TUIST_APP_CONFIG_TOKEN }}
+      - name: Generate Sparkle appcast
+        if: env.should-release == 'true'
+        working-directory: app
+        env:
+          TUIST_APP_PRIVATE_SPARKLE_KEY: ${{ secrets.TUIST_APP_PRIVATE_SPARKLE_KEY }}
+        run: echo "$TUIST_APP_PRIVATE_SPARKLE_KEY" | .build/artifacts/sparkle/Sparkle/bin/generate_appcast --link https://github.com/tuist/tuist/releases --download-url-prefix https://github.com/tuist/tuist/releases/download/${{ steps.next-version.outputs.NEXT_VERSION }}/app.dmg -o appcast.xml build/artifacts --ed-key-file -
+      - name: Commit changes
+        id: auto-commit-action
+        uses: stefanzweifel/git-auto-commit-action@v5
+        if: env.should-release == 'true'
+        with:
+          tagging_message: ${{ steps.next-version.outputs.NEXT_VERSION }}
+          commit_message: "[Release] Tuist App ${{ steps.next-version.outputs.NEXT_VERSION }}"
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v2
+        if: env.should-release == 'true'
+        with:
+          draft: false
+          repository: tuist/tuist
+          name: ${{ steps.next-version.outputs.NEXT_VERSION }}
+          tag_name: ${{ steps.next-version.outputs.NEXT_VERSION }}
```

Comment:
Shouldn't we prefix this one to disambiguate [here](https://github.com/asdf-community/asdf-tuist/blob/a2fabdd3be6fa2ff1a1d5d2a6b9c1a05cb03c491/lib/utils.bash#L20)?

Reply:
the `NEXT_VERSION` output already includes a prefix.